tls/s2n_change_cipher_spec.c

```diff
@@ -49,9 +49,6 @@ int s2n_client_ccs_recv(struct s2n_connection *conn)
     struct s2n_blob seq = {.data = conn->secure->client_sequence_number,.size = sizeof(conn->secure->client_sequence_number) };
     POSIX_GUARD(s2n_blob_zero(&seq));
 
-    /* Compute the finished message */
-    POSIX_GUARD(s2n_prf_client_finished(conn));
-
```

Comment:
I have a theory about why we did this in the CCS messages instead of the Finished message. I think it's to enforce "It is a fatal error if a Finished message is not preceded by a ChangeCipherSpec message at the appropriate point in the handshake." Like, as a defense-in-depth thing: even if we somehow massively mess up the handshake message parsing, if we haven't gotten a CCS message than the Finished message will fail.

Reply:
I re-added this line since its fine to keep this in the CCS message. Now the NPN handshake calls this function twice, instead of removing it from this message.